src/crimson/os/seastore/journal.h

```diff
@@ -107,6 +107,11 @@ class Journal {
   virtual ~Journal() {}
 
   virtual backend_type_t get_type() = 0;
+
+  virtual bool checksum_offloaded_to_device() {
+    // TODO: seastore currenlty does not support checksum offload for journaling
+    return false;
```

Comment:
Sorry, I didn't catch your comment here. Could you please let me know which simple implementation you mentioned is needed here?

Reply:
See https://github.com/ceph/ceph/pull/57782#discussion_r1677268711 and https://github.com/ceph/ceph/pull/57782#discussion_r1680319199.

For RBM, the journal's device is available from `CircularJournalSpace`. For segmented-journal, it can be reserved as "always need checksum".

Nit, let's be consistent about naming, probably `Journal::is_checksum_needed()` can be better?

Reply:
Thanks for the explanation. I updated the commit.